manifests/base/deploymentconfig.yaml

```diff
@@ -21,6 +21,9 @@ spec:
       containers:
         - name: elyra-aidevsecops-tutorial
           image: elyra-aidevsecops-tutorial
+          env:
+            - name: "APP_FILE"
+              value: "wsgi.py"
```

Comment:
I'm not sure if this will work. Do you want to run an API server? In that case, you'll need to have gunicorn in your dependencies and remove this line from the deployment config.

Reply:
> I'm not sure if this will work. Do you want to run an API server? In that case, you'll need to have gunicorn in your dependencies and remove this line from the deployment config.

Tested in thoth-test-core, otherwise the deployment gave me error that no app.py was found.

What changes using gunicorn respect to current solution?

Reply:
flask debug server is used to run the up this way, which is not nice for deployments (performance, security, ...). It's much better to use a WSGI server to run an app in a deployment.

Simply do `pipenv install gunicorn` and remove this environment variable. Commit `Pipfile` and `Pipfile.lock` and s2i tooling will do the rest for you to run an HTTP server.

Reply:
> flask debug server is used to run the up this way, which is not nice for deployments (performance, security, ...). It's much better to use a WSGI server to run an app in a deployment.
> 
> Simply do `pipenv install gunicorn` and remove this environment variable. Commit `Pipfile` and `Pipfile.lock` and s2i tooling will do the rest for you to run an HTTP server.

Thanks @fridex 🍻 💯